web/src/components/vis/VisTile.vue

```diff
@@ -1,5 +1,6 @@
 <script>
 import { format } from 'd3-format';
+import { downloadSVG } from '../../utils/exporter';
```

Comment:
I think we should prefer `@/utils/explorer' for file imports that don't have a need to be relative to the current file.

Reply:
see also https://github.com/girder/viime/pull/269#discussion_r330278495

Reply:
Seems that after we resolve what we want to do here, someone can go through and make sure @ is consistently used (or consistently not used). Don't think this blocks this PR.